app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -86,7 +86,14 @@ module(HELP_CONTENTS, [])
         <p>If you want to start from scratch, select "None".</p>
         <p>You can always edit the cluster configuration after you've created it.</p>`,
     'pipeline.config.expectedArtifact.matchArtifact': `
-        <p>This specifies which fields in your incoming artifact to match against. Every field that you supply will be used to match against all incoming artifacts. If all fields specified match, the incoming artifact is bound to your pipeline context.</p>
+        <p>
+          This specifies which fields in your incoming artifact to match against. If the artifact comes
+          from a pubsub message and a jinja template is specified for transforming those messages, then the fields
```

Comment:
I would make this less specific to pubsub/jinja, maybe just use that as an example. Maybe something like:

`The field comparisons are done against the incoming arifact. Example: if you are parsing artifacts from pub/sub messages via a jinja template, the comparison will be done against the after the pub/sub -> Spinnaker artifact translation.`

Thoughts?